src/core_server/internal/parsing/ceql_query/autogenerate_script.sh

```
@@ -1,42 +1,39 @@
+# Auto-generated by generate_antlr.py                          #
+# DO NOT MODIFY DIRECTLY, modify generate_antlr.py instead.    #
+#################################################################
+
+RED='\033[0;31m'
+GREEN='\033[0;32m'
+NORMAL_OUTPUT='\033[0m'
+
+# Work at the scripts directory
+cd "$(dirname "$0")"
+
+export ANTLR_HOME="~/.local/bin"
```

Shellcheck (0.11.0)

[warning] Tilde does not expand in quotes. Use $HOME.

(SC2088)